markdown/developers/training_guide.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Training material guide
+subtitle: A guide to writing instructions for training material.
+---
+
+# Contributing a training guide to nf-core
+
+Thank you for helping us write a guide to using an aspect of nf-core.
+
+## Links
+
+- [Questions](#questions)
+- [Objectives](#objectives)
+- [Glossary](#glossary)
+- [Before I begin](#before-i-begin)
+- [How to](#how-to)
+  - [Bytesize](#bytesize-talk)
+  - [Walkthrough](#walkthrough)
+- [Key points](#key-points)
+- [References](#references)
```

Comment:
The website will automatically generate a table of contents in the sidebar. Since we added that I have generally stopped having a manual ToC at the top like this, as it's easy to for the links to break..

Reply:
Ah, nice. I was hoping that was the case. I just wasn't sure.